src/components/Conversion/ConversionNOB.vue

```diff
@@ -133,6 +134,7 @@ export default class ConversionNOB extends Mixins(CommonMixin) {
 
   /** Called when user has clicked the Done button. */
   protected onDoneClicked (): void {
+    // eslint-disable-next-line no-undef
```

Comment:
Must be a new eslint warning. Is there an easy way to fix it? Thanks for the disable for now.

Reply:
The *.d.ts files may need to be reviewed. I previously (with [Vetur](https://vuejs.github.io/vetur/)) but no longer (with [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar)) had full context help with properties and methods. It would be nice to get that back.

Reply:
Might need some changes to get Volar working similar to AUTH-WEB 

https://github.com/bcgov/sbc-auth/blob/main/auth-web/volar.config.js

https://github.com/bcgov/sbc-auth/blob/f497bdee0ecb899645b1113f318f0bb25c4f9340/auth-web/package.json#L61

Reply:
Thanks, I'll try this ^^